go.mod

```diff
@@ -3,41 +3,41 @@ module github.com/cloudquery/filetypes/v4
 go 1.24.4
 
 require (
-	github.com/apache/arrow-go/v18 v18.3.1
+	github.com/apache/arrow-go/v18 v18.4.0
 	github.com/bradleyjkemp/cupaloy/v2 v2.8.0
 	github.com/cloudquery/codegen v0.3.31
-	github.com/cloudquery/plugin-sdk/v4 v4.87.3
+	github.com/cloudquery/plugin-sdk/v4 v4.87.4-0.20250723103024-0777f971931d
```

Comment:
I'll use the official SDK version once https://github.com/cloudquery/plugin-sdk/pull/2236 is released